Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,283 @@
+* Many pieces of software from Magix 
+
+[^pcgames_de_westerner2]: [The Westerner 2 im PC-Games-Test: "Ein enttäuschendes Adventure ... " ](https://www.pcgames.de/The-Westerner-2-Fenimore-Fillmores-Revenge-Spiel-38466/Tests/The-Westerner-2-im-PC-Games-Test-Ein-enttaeuschendes-Adventure-679282/)
+[^gamestar_de_westerner2]: [The Westerner 2: Westernheld Fenimore Fillmore ist er-
+wachsen geworden. Im Gegenzug
+schrumpft die Spielzeit auf Babyformat.](https://download.gamestar.de/public/43400/43452/05_2009_096.pdf)
+
+***
+
+## Supported Platforms:
+
+* light: Any (no software protection is employed, although it appears to mainly be marketed for Windows games regardless.)
+
+* professional: Windows
+(Some professional games are Hybrid PC/Mac discs, but the Mac versions of these games do not appear to utilize any sort of software protection.)[^108150]
+
+* audio: Audio CD
+
+***
+
+## Software Known to Detect:
+
+* ProtectionID
+* BinaryObjectScanner
+
+***
+
+## Preservation Instructions:
+
+light: None
+
+professional: Any sort of basic copy of the disc image appears to work.
+
+*** 
+
+## Associated File Attributes:
+
+### File Directories:
+
+#### "System":
+Directory for dummy files used by most professional discs so far, as well as some light[^Kenny] discs.
+
+#### "ZDAT":
+Directory for dummy files used by some, but not all light discs, and one known professional[^Petterson] disc so far. One calls it zdata[^108150], at least one uses System instead. Further variance likely.
+
+### File Names:
+Note: file names are not case sensitive, and vary from disc to disc.
+#### "gov_*.x64":
+Dummy file associated with professional. Names include gov_io.x64, gov_05.x64, and mov_io.x64.
+
+#### "iofile.x64":
+Dummy file associated with professional. Seems to be directly referenced by optgraph.dll.
+
+#### "sound.x64":
+Dummy file associated with professional. Seemingly not present on at least one professional disc.
+
+#### "optgraph.dll":
+Dll file responsible for software protection on professional discs.
+
+***
+
+## Known Manufacturing Information:
+
+### Known Manufacturing Plants:
+
+* optimal media production
+
+### Known Ringcode Information:
+
+#### Mastering Code:
+
+* manufactured by optimal media production	A######-0[Disc Number]
+* OH! Datenservice GmbH   www.oh-datenservice.de	A######-0[Disc Number]
+* KIDDINX	A######-0[Disc Number]
+
+#### Mastering SID Code:
+
+* IFPI L571
+
+#### Toolstamp or Mastering Code:
+
+* {Unordered list of toolstamp or mastering codes known to be associated with the DRM.}
+
+#### Mould SID Code:
+
+*  IFPI 97xx
+
+***
+
+# Reference Material:
+
+## Media: 
+
+Add example image to template is right, how do i put these in here
+1. [SoftICE for Windows 95 running before booting the game.](SoftICE_Detected.png)
+2. ["SoftICE for Windows 95" being detected.](SoftICE_Detected.png)
+this is from a specific game but idk how i put that on there iether
+***
+
+## Additional Resources:
+
+### Official Resources:
```

Comment:
Please add these resources as full links, you should be able to use the contents of the footnotes. For example, just add the following text as an item instead of the footnote:
* [The current optimal media website link.](https://www.optimal-media.com/)

Reply:
Fixed the formatting issue. If you also meant I needed to add more resources, let me know.